packages/oidf-client/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "@sphereon/ssi-sdk.oidf-client",
+	"version": "0.30.1",
+	"source": "src/index.ts",
+	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"veramo": {
+		"pluginInterfaces": {
+			"IOIDFClient": "./src/types/IOIDFClient.ts"
+		}
+	},
+	"scripts": {
+		"build": "tsc",
+		"build:clean": "tsc --build --clean && tsc --build",
+		"generate-plugin-schema": "ts-node ../../packages/dev/bin/sphereon.js dev generate-plugin-schema"
+	},
+	"dependencies": {
+		"openid-federation-modules-openapi": "link:../../../OpenID-Federation/build/js/packages/openid-federation-modules-openapi",
```

Comment:
@zoemaas 
Please use public packages now
https://www.npmjs.com/search?q=%40Sphereon%2FOpenID-Federation

Reply:
Fixed the issues with the public packages